package.json

```diff
@@ -19,14 +19,11 @@
     "test:config:react": "cd packages/eslint-config-godaddy-react && npm test",
     "test:config:react-flow": "cd packages/eslint-config-godaddy-react-flow && npm test",
     "test:config:typescript": "cd packages/eslint-config-godaddy-typescript && npm test",
-    "test:config:react-typescript": "cd packages/eslint-config-godaddy-react-typescript && npm test",
-    "prepush": "npm test"
-  },
-  "prepush": {
-    "tasks": [
-      "npm run prepush"
-    ]
+    "test:config:react-typescript": "cd packages/eslint-config-godaddy-react-typescript && npm test"
   },
+  "pre-commit": [
+    "test"
+  ],
```

Comment:
I'm not a fan of this, it discourages certain styles of development (e.g. TDD). why shouldn't I be able to commit locally with broken tests. Yeah I can always do `--no-verify` and commit anyway, but its a thing I need to remember. 

What exactly are we trying to prevent with this? direct commits to master with broken tests? PRs being created with broken tests (I actually think its useful sometimes to have this)?

Reply:
I don't really have a horse in this race, but I'm +1 to @msluther's feedback.

Reply:
that's fair, though tbh it was `prepush` before. I just moved away from `prepush` due to a security vuln. If we want to remove it I'm all for it and can probably be done in a follow up PR where we can get a full consensus on it.

Reply:
If we want to break it up, can we have another PR just drop it and try to merge that before this, rather than touching it to change what its doing int the GHActions PR?  I'm also okay just dropping it here if we get enough folks to weigh in on it.

Reply:
I agree with the approach of reverting this to its original behavior and having a follow-up PR to address it. This really isn't related to CI, so should be handled and discussed separately.

Reply:
https://github.com/godaddy/javascript/pull/130

Reply:
I'm trying to wrap my head around this. The argument is that it is better to rely on github to ensure that tests are run rather than local git because `--no-verify` is too onerous? Is that correct or am I misunderstanding?

Also, couldn't we just replace [pre-push](https://www.npmjs.com/package/pre-push) with [prepush](https://www.npmjs.com/package/prepush) instead of switching to pre-commit?